src/assets.rs

```diff
@@ -226,33 +222,71 @@ impl<'a> DerefMut for AssetEditor<'a> {
 
 /// Assets stored in an assets store.
 pub struct Assets {
+    /// A key-value store for assets.
     pub store: AssetsStore,
+
+    /// A set of asset codes loaded from verified asset libraries.
+    pub verified_assets: HashSet<AssetCode>,
 }
 
 impl Assets {
     #![allow(dead_code)]
 
-    /// Create assets.
+    /// Create unverified assets.
+    ///
+    /// To load verified assets afterwards, use `verify_asset`.
     pub fn new<L: Ledger>(store: AssetsStore) -> Result<Self, KeystoreError<L>> {
-        Ok(Self { store })
+        Ok(Self {
+            store,
+            verified_assets: HashSet::new(),
+        })
     }
 
     /// Iterate through the assets.
     pub fn iter(&self) -> impl Iterator<Item = Asset> + '_ {
         self.store.iter().cloned()
     }
 
-    /// Get the asset by the code.
+    /// Load a verified asset library with its trusted signer.
+    ///
+    /// Call this function at least once after assets resource is created.
```

Comment:
You don't _have_ to call this function. Only if you have a verified asset library that you want to include. Copying from the existing documentation (and tweaking a little):
```
Note that the verified status of assets is not persisted in order to
    /// preserve the verified library as the single source of truth about which assets are verified.
    /// Therefore, this function must be called each time an `Assets` store is created or opened in order to
    /// ensure that the verified assets show up in the assetlibrary.
```

Reply:
Thanks! I tweaked it a bit, removing `or opened` and changing `assetlibrarry` to `verified_assets`.